mbq/metrics/__init__.py

```diff
@@ -12,25 +12,22 @@
 
 logger = logging.getLogger('mbq.metrics')
 
+_constant_tags = {}
 _initialized = False
 _namespace = None
-_constant_tags = {}
-_statsd = utils.NullStatsd()
+_statsd = datadog.DogStatsd(
```

Comment:
This used to break the tests on local but now that they run in a container everything works ✨